### PR TITLE
enh(release): Add Centreon Web 21.04.3 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,38 @@ If you have feature requests or want to report a bug, please go to our
...
+### 21.04.3
+
+#### Enhancements
+
+- [Core] Implement API log mechanism
+- [Graph] Diverse export size options
+
+#### Bugfixes
+
+- [ACL] Display message if user doesn't have rights to access to a page
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:31]: We must remove this ticket, it had a bad fix version and is not fixed in this version

File: en/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,38 @@ If you have feature requests or want to report a bug, please go to our
...
+- [APIv2] Use poller's page ACL rights on Topology API endpoints
+- [Configuration] InfluxDB configuration columns are deleted in Broker from
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:33]: I think it's 'form' and not 'from'

File: fr/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,38 @@ If you have feature requests or want to report a bug, please go to our
...
+#### Améliorations
+
+- [Core] Implement API log mechanism
+- [Graph] Diverse export size options
+
+#### Correctifs
+
+- [ACL] Display message if user doesn't have rights to access to a page
+- [APIv2] Use poller's page ACL rights on Topology API endpoints
+- [Configuration] InfluxDB configuration columns are deleted in Broker from
```
Comment (mdebauchez) [inline fr/releases/centreon-core.md:33]: I think it's 'form' and not 'from'

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,37 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.3
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:23]:
```suggestion

`June 28, 2021`

```

File: fr/releases/centreon-core.md [hunk 5]
```diff
@@ -19,6 +19,37 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.3
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:23]:
```suggestion

`28 juin 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Pull request merged**